pallets/root-of-trust/src/lib.rs

```diff
@@ -21,8 +21,9 @@
 //! This module implements a Root Of Trust linked to a `membership` or `tcr` pallet which
 //! can be used to let entities represented by their `AccountId` manage certificates
 //! and off-chain certificates in Public Key Infrastructure fashion (SSL / TLS like).
-
+#[cfg(feature = "runtime-benchmarks")]
```

Comment:
do you need that flag? It is normally showing up on top of the `benchmarking.rs` file

Reply:
Indeed not required, removed, Done :+1: